### PR TITLE
docker-compose: set image to orbin/simh-vax

### PR DESCRIPTION
**Pull request by alkhimey**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   simh:
-    build: .
```
Comment (OrBin) [inline docker-compose.yml:4]: Can you leave it commented? when changing the dockerfile, we'd like to use this line for building automatically.

**Review by OrBin — CHANGES_REQUESTED** [review]

**Review by OrBin — APPROVED** [review]

**Comment by alkhimey** [comment]
Sorry did not have time to do it in the last couple of days. I guess you will sort it out yourself 😉
Don't forget about the issue with changing the root directory from `simh-vax-docker` to `vax`.

**Comment by OrBin** [comment]
Yeah, I've added a `build.sh` script to handle building

**Comment by OrBin** [comment]
Thanks again!

**Pull request merged**